docs/control-structures.rst

```diff
@@ -141,11 +187,8 @@ the gas can be specified with special options ``.value()`` and ``.gas()``, respe
 The modifier ``payable`` has to be used for ``info``, because otherwise, the `.value()`
 option would not be available.
 
-Note that the expression ``InfoFeed(addr)`` performs an explicit type conversion stating
-that "we know that the type of the contract at the given address is ``InfoFeed``" and
-this does not execute a constructor. Explicit type conversions have to be
-handled with extreme caution. Never call a function on a contract where you
-are not sure about its type.
+...note::
+    The expression ``InfoFeed(addr)`` performs an explicit type conversion stating that "we know that the type of the contract at the given address is ``InfoFeed``" and this does not execute a constructor. Explicit type conversions have to be handled with extreme caution. Never call a function on a contract where you are not sure about its type.
```

Comment:
Actually nowadays, the proper way to do this would be to use
```
function setFeed(InfoFeed addr) public { feed = addr; }
```
but then of course we cannot have this warning here anymore.

Reply:
@chriseth Huh, I just formatted this, content wasn't from the FAQ, could job this found it anyway…

Reply:
Think I'll create a separate issue for this.